test/integration/features/cors.feature

```diff
@@ -0,0 +1,23 @@
+@cors
+Feature: Cors Add-on 
+  As Minishift user I can install and apply Cors add-on from minishift-addons repository.
+
+  Scenario: Installing the add-on
+     When file from "https://raw.githubusercontent.com/minishift/minishift-addons/master/add-ons/cors/cors.addon" is downloaded into location "download/cors"
+      And executing "minishift addons install ../../testing/integration-test/download/cors" succeeds
+     Then stdout should contain "Addon 'cors' installed"
```

Comment:
This does not look intuitive, especially when we are in the addons repository.  We need a way to point to the relative path for the addon. We want to test the PR alos i.e. the add-on would be in the PR itself. So it won't have a physical location in the repository itself.

Reply:
@LalatenduMohanty +1, will create an issue in the Minishift

Reply:
@LalatenduMohanty Haha, just realized I can do: `When executing "minishift addons install ../../add-ons/cors" succeeds`. No need for any copying, any download, just dead simple. Thanks for pointing it out, would not realize this.